server/routes/script.js

```diff
@@ -1,9 +1,59 @@
 const express = require('express');
 const router = express.Router();
+const axios = require('axios');
+
+const url = process.env.api;
```

Comment:
Parse the environment variables and put them into a single file.

config.js

const dotenv = require('dotenv').config();

export const themoviedb = {
    host: process.env.THE_MOVIE_API_HOST,
}